.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      # Keep the experimental modules up-to-date
+      - dependency-name: "github.com/grafana/xk6-*"
+        dependency-type: "all"
+    target-branch: "master"
```

Comment:
Nit; I think this is not strictly required because it automatically looks for the default branch, and so if we don't add it, we don't need to change it if anytime we in the future we change the default branch. But it's also true that in such case, this will be the least worrying thing, so up to you!

Reply:
Good point, will drop this one.